code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -365,6 +365,7 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 		/obj/effect/essentials_set/mimir = list(CAT_ARMMOD, "Mark 1 Mimir Resistance set", 0,"black"),
 		/obj/item/armor_module/attachable/tyr_extra_armor/mark1 = list(CAT_ARMMOD, "Mark 1 Tyr extra armor module", 0,"black"),
 		/obj/item/armor_module/attachable/better_shoulder_lamp/mark1 = list(CAT_ARMMOD, "Mark 1 Baldur light armor module", 0,"black"),
+		/obj/effect/essentials_set/vali = list(CAT_ARMMOD, "Vali chemical enhancement set", 0,"black"),
```

Comment:
Careful with making it free

Reply:
People claim for it to be on the weak side currently. I added some usability optimizations and will see how it goes next testmerge.

Reply:
First test people said it was on the weak side. I made some usability fixes and will see how it goes next testmerge

Reply:
Aye hence why I said be careful